From d97993a7a03f7809ca5e41b61a26b4cc6de379d8 Mon Sep 17 00:00:00 2001
From: Tomas Mraz <tomas@openssl.org>
Date: Thu, 19 Oct 2023 10:30:15 +0200
Subject: [PATCH 373/381] load_key_certs_crls(): There is no quiet argument

This fixes broken cherry-pick from the master branch
where there is a quiet argument.

Reviewed-by: Matt Caswell <matt@openssl.org>
Reviewed-by: Tom Cosgrove <tom.cosgrove@arm.com>
Reviewed-by: Richard Levitte <levitte@openssl.org>
(Merged from https://github.com/openssl/openssl/pull/22434)

(cherry picked from commit 307048cd4e887de688eb71af713c64962261cd29)
---
 apps/lib/apps.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/apps/lib/apps.c b/apps/lib/apps.c
index 2c301f3ca8..a632b0cff2 100644
--- a/apps/lib/apps.c
+++ b/apps/lib/apps.c
@@ -961,13 +961,11 @@ int load_key_certs_crls_suppress(const char *uri, int format, int maybe_stdin,
                                  params, NULL, NULL);
     }
     if (ctx == NULL) {
-        if (!quiet)
-            BIO_printf(bio_err, "Could not open file or uri for loading");
+        BIO_printf(bio_err, "Could not open file or uri for loading");
         goto end;
     }
     if (expect > 0 && !OSSL_STORE_expect(ctx, expect)) {
-        if (!quiet)
-            BIO_printf(bio_err, "Internal error trying to load");
+        BIO_printf(bio_err, "Internal error trying to load");
         goto end;
     }
 
-- 
2.34.1

